mmrm_R_package.qmd

```diff
@@ -0,0 +1,33 @@
+---
+title: "MMRM R Package Development"
+---
+
+**Lead**: Daniel Sabanes Bove (Roche)
+
+**Objective**: To develop a comprehensive R package for mixed models with repeated measures (MMRM) that is robust, well documented and thoroughly tested
+
+```{r setup, include=FALSE}
+## load packages
+packs.html <- c("knitr", "pander", "reporttools", "dplyr", "readxl")
+for (i in 1:length(packs.html)){library(packs.html[i], character.only = TRUE)}
+
+knitr::opts_chunk$set(echo = TRUE)
+panderOptions('table.alignment.default', function(df){ifelse(sapply(df, is.numeric), 'right', 'left')})
+
+path <- getwd()
+
+## member list
+members <- as.data.frame(read_excel(paste(path, "/data/members.xlsx", sep = ""), col_types = NULL))
```

Comment:
seems this code is duplicate from the one on the people page, I would recommend to write a small helper function and then use that in both places (as we will need it for other task forces again too)